### DISCUSSION TITLE
Webhook settings [Jul 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6107] [Discord Threads]
> 🙋‍♂️ **j0**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@1070958642997755934> can you provide more information about your setup? you can copy project information from project menu here  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1122174278335463425)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
